install_linux.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+
+set -euxo pipefail
```

Comment:
While this is technically a bit better (thought `-x` is no good), it has no meaningful behavioral impact. `bash` is already invoked with `-e` via the shebang in the original version.